Libraries/xcexecution/Sources/NinjaExecutor.cpp

```diff
@@ -114,6 +114,12 @@ NinjaHash(std::string const &input)
     return ss.str();
 }
 
+static std::string
+NinjaHash(std::string const &input)
```

Comment:
What's the point of this? From what I can see, the only caller for these functions actually calls the `Internal` version. Also, keeping the signature the same and not having an extra wrapper seems pretty easy in this case.